ystafdb/bin/ystafdb.py

```diff
@@ -3,22 +3,43 @@
 """YSTAFDB CLI
 
 Usage:
-  ystafdb-cli regenerate <dirpath>
+  ystafdb-cli -i <input/dirpath> -o <output/dirpath> -f <format>
 
 Options:
   -h --help     Show this screen.
   --version     Show version.
 
 """
+import argparse
```

Comment:
I think my comment was not clear.
I meant to say that I would have liked to use `docopt` instead of argparse. But we should remove the import of `docopt` if we are not using it anymore.